2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,8 +18,18 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let key;
+  let value;
+  let countryCurrencyObject = {};
+  for (let i = 0; i < countryCurrencyCodes.length; i++) {
+   key = countryCurrencyCodes[i][0];
+   value = countryCurrencyCodes[i][1];
+  countryCurrencyObject[key] = value;
 }
+return countryCurrencyObject
+}
+
+console.log(createLookup(COUNTRY_CURRENCY_CODES))
```

Comment:
Comment: Nice! 

Consider: You could consider to check a .forEach 
https://www.w3schools.com/jsref/jsref_foreach.asp 
  const lookup = {};
  countryCurrencyCodes.forEach(([country, currency]) => {
    lookup[country] = currency;
  });
  return lookup;
}

const lookup = createLookup(COUNTRY_CURRENCY_CODES);
console.log(lookup);